deepview_profile/analysis/request_manager.py

```diff
@@ -49,14 +47,14 @@ def _handle_analysis_request(self, analysis_request, context):
         start_time = time.perf_counter()
         try:
             logger.debug(
-                'Processing request %d from (%s:%d).',
+                "Processing request %d from (%s:%d).",
                 context.sequence_number,
                 *(context.address),
             )
-            connection = self._connection_manager.get_connection(
-                context.address)
+            connection = self._connection_manager.get_connection(context.address)
             analyzer = analyze_project(
-                connection.project_root, connection.entry_point, self._nvml)
+                connection.project_root, connection.entry_point, self._nvml,analysis_request.ddp_analysis_request
```

Comment:
Please format this line with a code formatter.

Reply:
Moved parameters into different lines.